handlers/commands/index.js

```diff
@@ -15,8 +15,17 @@ module.exports = function(models) {
         modules.push(require("./"+file));
     });
 
+    commandNames = [];
+
     modules.forEach(commandModule => {
-        handlerList.push(new commandModule(models))
+        handlerList.push(new commandModule(models));
+    });
+
+    handlerList.forEach(handler => {
+        if (!commandNames.includes(handler.name.toLowerCase()))
+            commandNames.push(handler.name.toLowerCase());
+        else
+            throw (handler.name + " was detected as a duplicate command");
     });
```

Comment:
After all handlers have been instantiated, the code iterates over them and builds an array of their names (lowercase). If it detects a duplicate, an error is thrown and the bot won't start